src/library/scala/collection/mutable/ArrayBuffer.scala

```diff
@@ -64,6 +64,13 @@ class ArrayBuffer[A] private (initialElements: Array[AnyRef], initialSize: Int)
     size0 = n
   }
 
+  /** Shrinks the ArrayList to one that is adjusted to the current number of elements in it,
```

Comment:
I find this comment a bit misleading. I would expect the method to truncate the array to the currently needed size but what it really does is truncate to the next larger "normal" size (or not at all when < 8 elements).